account_avatax/models/account_move.py

```diff
@@ -151,7 +151,7 @@ def _avatax_compute_tax(self, commit=False):
             doc_type,
             self.partner_id,
             self.warehouse_id.partner_id or self.company_id.partner_id,
-            self.partner_shipping_id or self.partner_id,
+            self.shipping_add_id or self.partner_id,
```

Comment:
There is no shipping_add_id in account.move object, please change it to tax_address_id

Reply:
Sorry, I got confused with the tax_address_id definition in sale.order

@dreispt We could follow the same naming convention across account_avatax and account_avatax_sale

Reply:
Bad names are bad code. Done as suggested